### PR TITLE
Fixes #907 #908 #917 #918 #920 #921

### LINKED ISSUE #907: Variables are ignored in GraphQL subscriptions
### Describe the bug
The query variables sent in the request for GraphQL subscription over WebSockets are ignored. If we specify some variables in the GraphQL query of subscription, then we get this error - `Scope not present`

### Expected behaviour
The GraphQL query variables should be used by Space Cloud.

### Actual behaviour 
Space Cloud gives an error

### Your environment
- Space Cloud version: `v0.17.0`

>If this bug restricts your use of space-cloud, give it a thumbs up reaction, so that we can determine which bugs need to be fixed immediately.
>👍

### PR DESCRIPTION
**Pull request by YourTechBud**

### CODE REVIEW

File: gateway/utils/handlers/websocket.go [hunk 1]
```diff
@@ -125,14 +129,11 @@ type graphqlMessage struct {
...
-	Error            []gqlError  `json:"error,omitempty"`
-	Data             interface{} `json:"data,omitempty"`
-}
-
-type token struct {
-	Token string `json:"token,omitempty"`
+	Query     string                 `json:"query,omitempty"`
+	Token     string                 `json:"authToken"`
+	Variables map[string]interface{} `json:"variables"`
+	Error     []gqlError             `json:"error,omitempty"`
```
Comment (YourTechBud) [inline gateway/utils/handlers/websocket.go:135]:
```suggestion
	Error     []gqlError             `json:"errors,omitempty"`
```

File: gateway/config/config.go [hunk 2]
```diff
@@ -173,8 +173,8 @@ type StaticRoute struct {
 type Eventing struct {
 	Enabled       bool                    `json:"enabled" yaml:"enabled"`
 	DBAlias       string                  `json:"dbAlias" yaml:"dbAlias"`
-	Rules         map[string]EventingRule `json:"rules,omitempty" yaml:"rules"`
-	InternalRules map[string]EventingRule `json:"internalRules,omitempty" yaml:"internalRules,omitempty"`
+	Rules         map[string]EventingRule `json:"triggers,omitempty" yaml:"triggers"`
+	InternalRules map[string]EventingRule `json:"internalSecurityRules,omitempty" yaml:"internalSecurityRules,omitempty"`
```
Comment (Jayesh333) [inline gateway/config/config.go:177]: these are `internalTriggers` if I am right?

Comment (YourTechBud) [inline gateway/config/config.go:177]: ### 
```suggestion
	InternalRules map[string]EventingRule `json:"internalTriggers,omitempty" yaml:"internalTriggers,omitempty"`
```

**Comment by YourTechBud** [comment]
@Jayesh333 i made the changes. Approve and merge the pr once you are done

**Pull request merged**